app/DAO/BakeryDatabase.scala

```diff
@@ -0,0 +1,38 @@
+package DAO
+
+import javax.inject.Inject
+import play.api.db.Database
+
+import scala.concurrent.Future
+import models.DatabaseExecutionContext
+
+class BakeryDatabase @Inject() (
+    db: Database,
+    databaseExecutionContext: DatabaseExecutionContext
+) {
+
+  def getDatabaseName: Future[String] = {
```

Comment:
Rename to `getDataBaseTables`, be accurate about the purpose of your functions

Reply:
renamed method to `getDatabaseTables`, accordingly made changes to the APIController method call